ci/snap/snapcraft.yaml

```diff
@@ -0,0 +1,29 @@
+name: git-machete2
```

Comment:
Is this file picked up now by the test CI script? 🤔 or rather snap/snapcraft.yaml

Reply:
@PawelLipski Yes, it is picked up

Reply:
Hmm, turns out it wasn't picked up, i didn't even know that there already is snapcraft.yaml file in repo :O